### ISSUE TITLE
File user-config.h does not exist

### ISSUE THREAD

**Issue by wimjan1** [SWAT-engineering/kiwi-air-monitor#40]
I have tried to upload your code via Platform IO but it states that the file user-config.h does not exist.

is there something I am missing?

**Comment by wimjan1** [comment]
I have found the user-config-template.h and have saved it as user-config.h in the same folder( software\include)

with this my question has been resolved but I would like to ask if this can be added to the user manual.

**Comment by DavyLandman** [comment]
yes, that is the solution, and you can now fill in details that are relevant for your setup (such as mqtt & wifi connection settings if you need them).

To be honest, I want to move away from this model, see #36. Where we publish binaries that anyone can flash, and with a push of a dedicated button, you can trigger the config mode to connect a wifi accespoint and enter your details. but I have to find time for that first.
